app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -930,8 +930,17 @@ DeviceDelegateOpenXR::SetReorientTransform(const vrb::Matrix& aMatrix) {
 }
 
 void
-DeviceDelegateOpenXR::Reorient(vrb::Matrix& transform) {
-  m.reorientMatrix = DeviceUtils::CalculateReorientationMatrixWithoutRoll(transform, GetHeadTransform().GetTranslation());
+DeviceDelegateOpenXR::Reorient(vrb::Matrix &transform, ReorientMode mode) {
```

Comment:
nit: the space must be before the "&"

As said before, it seems that "transform" can be "const"

Reply:
OK